contributors/devel/development.md

```diff
@@ -279,20 +279,42 @@ To build binaries for all platforms:
 make cross
 ```
 
+#### Install etcd
+```
+hack/install-etcd.sh
+```
+
+#### Verification
+```
+# Run all hack/verify-*.sh scripts
+make verify
+```
+
 #### Test
 
 ```sh
 cd $working_dir/kubernetes
 
+# Run all hack/update-*.sh scripts
+# For small change, run "make verify" and then relevant scripts, for example:
+# hack/update-gofmt.sh (when adding new files)
+# hack/update-bazel.sh (when updating dependencies / imports)
+make update
```

Comment:
`make verify` (mentioned above) would have failed if anything isn't updated. That is to say, contributors would have already fixed/updated everything if they follow the workflow in this doc. 

`make update` can be combined with the verification section. Contributors either `make verify` and then fix each individual failure (by running hack/update-xxx), or just `make update` and don't need to `make verify` (otherwise `make update` isn't doing the right thing).